src/main/java/gov/nasa/pds/api/engineering/elasticsearch/ElasticSearchUtil.java

```diff
@@ -86,9 +96,12 @@ static public ProductWithXmlLabel ESentityProductToAPIProduct(EntityProduct ep)
 		}
 		*/
 		
+		ArrayList<Reference> investigations = new ArrayList<Reference>();
+		ArrayList<Reference> osc = new ArrayList<Reference>();
```

Comment:
I prefer a more explicit variable name, obsSystemComponents for example of observationSystemComponents since this line is far away from the one where the observation system components are added. I am picky. You can also more the deplaration next to where the elements are added (lines 127, 128), then it will be more readable what osc is about

Reply:
Both option work for me.

Reply:
Done.